### DISCUSSION TITLE
{{ Created By.0.firstName }} handlebar error???

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#4437] [Help]
Hello,

I try to create a formula field that shows the fisrtname of the employee.

But I have an handlebar error... The button is disabled.

Any idea?

**Comment by mjashanks** [comment]
You don't have access to the full user details in here.

Your only options are `{{ Created By.0.primaryDisplay }}` and `{{ Created By.0._id }}`

So, you could go to your users table and 
- update the display column to firstName, then use primaryDisplay
OR
- add a formula field to your user table called "Full Name"... to output "Shanks, Michael", then in your front end you could use Javascript to `return $("Created By")[0].primaryDisplay.split(",");`

**Reply by oswaldB** [reply]
Hello, 

It doesn't work.


![Screenshot from 2022-02-11 19-45-04](https://user-images.githubusercontent.com/3023526/153659290-8eb38984-b775-4330-8a3f-d208a88131ad.png)

**Reply by mjashanks** [reply]
Oh sorry i didn't realise that you were using it in a formula. That looks like it could be a bug.

I was using it in the Design section, so was actually like `{{ Repeater.Table.Created By.0._id }}`

**Reply by mjashanks** [reply]
I'm not 100% if we should be able to use Created By in a formula field.

**Reply by oswaldB** [reply]
OK so why it is listed on the left? Because originally i just click on it ;)

**Reply by mjashanks** [reply]
well yes - thats why I say it looks like a bug!

**Reply by oswaldB** [reply]
oki :)
